umodbus/client/tcp/__init__.py

```diff
@@ -0,0 +1 @@
+from .tcp import *
```

Comment:
why not move the `tcp.py` file contents to this `__init__.py`? Saves an import and an extra file

Reply:
I was tempted to do it but I noticed in the project all `__init__.py` files are empty so I aimed for consistency.
I move the contents of `tcp.py` then to `__init__.py`